packages/swarm-components/lib/Checkbox.js

```diff
@@ -34,4 +34,39 @@ const Checkbox = props => {
   }), React.createElement("span", null, label || children));
 };
 
-export default Checkbox;
\ No newline at end of file
+export default Checkbox;
+Checkbox.__docgenInfo = {
+  "description": "",
+  "methods": [],
+  "displayName": "Checkbox",
+  "props": {
+    "checked": {
+      "required": true,
+      "flowType": {
+        "name": "boolean"
+      },
+      "description": ""
+    },
+    "disabled": {
+      "required": false,
+      "flowType": {
+        "name": "boolean"
```

Comment:
I'm confused - name is a boolean?

Reply:
I don't really understand how these objects for the components work, does it have something to do with `react-docgen` ?

Reply:
Yes, it's part of `react-docgen` to be used in the props sections of the pages.

Reply:
Could you add something about react-docgen in the readme with a link to their documentation?